lang/en.json

```diff
@@ -107,6 +107,8 @@
     "GMTOOLKIT.Settings.SessionEnd.ExportChat.name" : "Export Chat at Session End",
     "GMTOOLKIT.Settings.SessionEnd.ExportChat.hint" : "Prompt to save the chat log when running the End of Session Routine macro.",
 
-    "GMTOOLKIT.Message.SessionEnd.NoPermission" : "Only GMs can administer end of session rites!"
+    "GMTOOLKIT.Message.SessionEnd.NoPermission" : "Only GMs can administer end of session rites!",
 
+    "GMTOOLKIT.CHAT.LittlePrayerResult" : "<p>{actorName} offered a Little Prayer to the Gods ({littlePrayerResult}).</p>",
+    "GMTOOLKIT.CHAT.StatusChange" : "<p>{status} changed from {originalStatus} to {newStatus} for {targetName}.</p>"
```

Comment:
This appears equivalent to `GMTOOLKIT.TokenHudExtension.StatusChanged` (line 16).  Also can't see that it's being used? What am I missing? Same applies to fr.json and pl.json.